recipes/vcctl/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "vcctl" %}
+{% set version = "1.6.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/volcano-sh/volcano/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3525856b84b3cd1e58e335f55d45fd60d4c02a1c8d98944b010e6c0518d4155a
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+    - make >=4.3
+  host:
+  run:
+
+test:
+  commands:
+    - vcctl
+
+about:
+  home: https://volcano.sh/en/docs/cli/
+  summary: A CLI for Volcano
+  description: |
+      A Command Line Interface (CLI) to manage resources of Volcano.
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
Please do not check tasks on the TODO list you have not completed. This GO binaries has a bunch of statically linked dependencies. You need to collect the licenses of these other packages into a file and add this file to the license_file section.

There is a tool which can do collect the licenses for you automatically.

https://github.com/conda-forge/conda-forge.github.io/issues/1052

Reply:
Thanks for pointing this out. Will do!